tests/test_core.py

```diff
@@ -1538,44 +1535,7 @@ def test_too_many_columns_detection(engine_testaccount, db_parameters):
     metadata.create_all(engine_testaccount)
     inspector = inspect(engine_testaccount)
     # Do test
-    original_execute = inspector.bind.execute
-
-    def mock_helper(command, *args, **kwargs):
-        if "_get_schema_columns" in command:
-            # Creating exception exactly how SQLAlchemy does
-            raise DBAPIError.instance(
-                """
-            SELECT /* sqlalchemy:_get_schema_columns */
-                   ic.table_name,
-                   ic.column_name,
-                   ic.data_type,
-                   ic.character_maximum_length,
-                   ic.numeric_precision,
-                   ic.numeric_scale,
-                   ic.is_nullable,
-                   ic.column_default,
-                   ic.is_identity,
-                   ic.comment
-              FROM information_schema.columns ic
-             WHERE ic.table_schema='schema_name'
-             ORDER BY ic.ordinal_position""",
-                {"table_schema": "TESTSCHEMA"},
-                ProgrammingError(
-                    "Information schema query returned too much data. Please repeat query with more "
-                    "selective predicates.",
-                    90030,
-                ),
-                Error,
-                hide_parameters=False,
-                connection_invalidated=False,
-                dialect=SnowflakeDialect(),
-                ismulti=None,
-            )
-        else:
-            return original_execute(command, *args, **kwargs)
-
-    with patch.object(inspector.bind, "execute", side_effect=mock_helper):
```

Comment:
This mock was actually unused in the test, as `engine.execute` was never called. Since `engine.execute` no longer exists in 2.0, I removed it.